target/linux/ath79/dts/qca9563_tplink_eap245-v3.dts

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	compatible = "tplink,eap245-v3", "qca,qca9563";
+	model = "TP-Link EAP245 v3";
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &led_green;
+		led-failsafe = &led_orange;
+		led-running = &led_green;
+		led-upgrade = &led_orange;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_green: led_green {
+			label = "tp-link:green:power";
+			gpios = <&gpio 7 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_orange: led_orange {
+			label = "tp-link:orange:failsafe";
+			gpios = <&gpio 9 GPIO_ACTIVE_HIGH>;
+		};
```

Comment:
I'd refer to these as status LEDs, leading to the following:
```
		led_status_green: status_green {
			label = "tp-link:green:status";
			gpios = <&gpio 7 GPIO_ACTIVE_HIGH>;
			default-state = "on";
		};

		led_status_orange: status_amber {
			label = "tp-link:amber:status";
			gpios = <&gpio 9 GPIO_ACTIVE_HIGH>;
		};
```

The default-state will have the LED on earlier in the boot process, i.e. before scripts like the one evaluating led-aliases will work.

Reply:
I didn't really like the 'failsafe' name for the amber LED either, but couldn't think of anything more appropriate. Thanks for the suggestion.